### DISCUSSION TITLE
POV : Deepin, Wine and Windows driver.

### DISCUSSION THREAD

**Discussion by KevinGIRAULT** [linuxdeepin/developer-center#6725] [General | 常规讨论]
Linux, thanks in particular to Deepin and UOS, is becoming more and more enjoyable, even more enjoyable than Windows for users who have never touched Redmond's OS.

However, when we have slightly exotic hardware, that's when the problems start. My son has a toy globe that needs updating. But Windows is mandatory. Wine? It doesn't support peripherals.

As long as this is the case, Linux may not penetrate the market any further than it is now on the desktop, and Windows may continue to reign supreme.

I don't know what Deepin could do, but here's the reality, IMO, from which there's no escape.

**Comment by BLumia** [comment]
The manufactory of the toy should try to support platforms other than Windows, that's probably nothing we can do...

**Comment by raspin0** [comment]
You shouldn't expect everything made for windows to work on deepin. It's a different operating system after all. The support and software for this globe is provided by its manufacturer, you should go ask them.
